docs/api.md

```diff
@@ -0,0 +1,21 @@
+# Antrea API
+
+This document lists all the API group versions currently or previously supported
+by Antrea, along with information related to their deprecation and removal when
+appropriate. it is kept up-to-date as we evolve the Antrea API.
+
+For information about the Antrea API versioning policy, please refer to this
+[document](versioning.md).
+
+| API group | API version | API Service? | Introduced in | Deprecated in | Removed in |
```

Comment:
`Removed (planned for removal) in`? I'm okay with it as is if it's common practice to write this way.

Reply:
Or maybe moving the removed API to a separate table and use "Removed in" there, here it could use "Planned removal for", so that the table for existing API won't get too big finally and people can have a clear view of supported ones.